generators/app/gulp-tasks/build.js

```diff
@@ -0,0 +1,85 @@
+**
+ * @license
+ * Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+ */
+
+'use strict';
+
+const gulp = require('gulp');
+const gulpif = require('gulp-if');
+const imagemin = require('gulp-imagemin');
+const mergeStream = require('merge-stream');
+
+const polymer = require('polymer-build');
+const PolymerProject = polymer.PolymerProject;
+const fork = polymer.forkStream;
+const addServiceWorker = polymer.addServiceWorker;
```

Comment:
@abdonrd lets stick with this method for now, It has been tested and works fine. Once we add this repo to NPM Registry it will be easier to test these types of changes. Same goes for `module.export`.